### PR TITLE
Add Test For 679636490

### PR DESCRIPTION
**Pull request by droogmic**
Combining bus:lanes and lanes:bus

### CODE REVIEW

**Comment by droogmic** [comment]
Taken from discussion in https://github.com/a-b-street/osm2lanes/issues/136

File: data/tests.yml [hunk 1]
```diff
@@ -309,7 +309,8 @@
 # incorrect speed limit
 - way_id: 102917976
   description: Road with impossible speed limit
-  rust: false
```
Comment (dabreegster) [inline data/tests.yml:312]: Ah, do you have a process for auto-formatting this file?

**Review by dabreegster — APPROVED** [review]

File: data/tests.yml (hunk 1 above)
Comment (droogmic) [reply data/tests.yml:312]: I... don't know.
VS code sometimes fiddles with the yaml I write but I don't know when or why :/

It's definitely not something I would know how to integrate with the CI (without investigating)

File: data/tests.yml [hunk 2]
```diff
@@ -309,7 +309,8 @@
...
+  rust:
+    false
```
Comment (droogmic) [inline data/tests.yml:313]:
```suggestion
  rust: false
```

File: data/tests.yml (hunk 1 above)
Comment (droogmic) [reply data/tests.yml]: I reverted that line for now

**Pull request merged**